app/src/main/java/rkr/simplekeyboard/inputmethod/latin/settings/SettingsActivity.java

```diff
@@ -58,7 +58,7 @@ public void onClick(DialogInterface dialog, int id) {
                     dialog.dismiss();
                 }
             });
-            builder.setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
+            builder.setNegativeButton(R.string.setup_cancel, new DialogInterface.OnClickListener() {
```

Comment:
`"OK"` should also be translatable in such case.

Reply:
It is your decision but I'm never translating 'OK'.
I wouldn't even know how ;)